package.json

```diff
@@ -7,54 +7,54 @@
     "renovate-config-validator": "dist/config-validator.js"
   },
   "scripts": {
-    "build": "run-s clean generate:* compile:* create-json-schema",
-    "build:docs": "run-s \"release:prepare {@}\" --",
+    "build": "run-s clean 'generate:*' 'compile:*' create-json-schema",
+    "build:docs": "run-s 'release:prepare {@}' --",
     "clean": "rimraf dist tmp",
     "clean-cache": "node tools/clean-cache.mjs",
     "compile:ts": "tsc -p tsconfig.app.json",
     "config-validator": "ts-node lib/config-validator.ts",
-    "create-json-schema": "ts-node tools/generate-schema.ts && prettier --write --cache \"renovate-schema.json\"",
-    "debug": "cross-env NODE_OPTIONS=--inspect-brk ts-node lib/renovate.ts",
+    "create-json-schema": "ts-node tools/generate-schema.ts && prettier --write --cache 'renovate-schema.json'",
+    "debug": "NODE_OPTIONS=--inspect-brk ts-node lib/renovate.ts",
     "doc-fix": "run-s markdown-lint-fix prettier-fix",
     "doc-fence-check": "node tools/check-fenced-code.mjs",
     "lint-documentation": "jest --coverage false test/documentation.spec.ts",
     "eslint": "eslint . --cache --cache-location .cache/eslint --report-unused-disable-directives",
```

Comment:
I don't know _why_ but ESLint takes _forever_ to start on GitHub Codespaces, both with `pnpm` and `yarn`. Do you have the same problem?

By the way that Jest error message that I had when starting the Codespace is now gone. So at least we have that. 😄

Reply:
yes, eslint is also taking a long time locally. I think it's unrelated to this PR. need to check main branch

Reply:
confirmed, it's also slow on main without cache 😕 I think it's related to the typescript-eslint v6 upgrade.